docs/setup/developer.md

```diff
@@ -49,7 +76,7 @@ There are also integration tests in `PIXL/test/` directory that can be run using
 ### Workflow
 
 Before raising a PR, make sure to **run all tests** for each PIXL module
-and not just the component you have been working on as this will help us catch unintentional regressions without spending GH actions minutes :-)
+and not just the component you have been working on as this will help us catch unintentional regressions without spending GH actions minutes.
```

Comment:
aw no smiley

Reply:
Sorry, we can always use some [emojis](https://gist.github.com/rxaviers/7360908) in the sections names and removed that one as it was at the end of the sentence and in the middle of the readme file